admin/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+</head>
+<body>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
```

Comment:
Should we add the integrity attribute from unpkg. ie https://unpkg.com/netlify-cms@2.10.23/dist/netlify-cms.js?meta

Reply:
Loading via npm instead